core/src/main/resources/hudson/model/UsageStatistics/global.properties

```
@@ -24,6 +24,8 @@ statsBlurb=\
   Help make Jenkins better by sending anonymous usage statistics and crash reports to the Jenkins project
 statsBlurbFIPS=\
   Help make Jenkins better by sending telemetry to the Jenkins project
+statsDescriptionFIPS=\
+  General usage statistics are not submitted when FIPS-140 compliance is requested.
```

```suggestion
  General usage statistics are never submitted when FIPS-140 compliance is requested.
```

To clarify it's independent of this setting?

---

I'm OK with either word, though I didn't infer the independence of the property from the change of wording.  My reading treated those two words as synonyms in that context.